roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-hpp.yml

```diff
@@ -71,6 +71,22 @@
           nodeSelector:
             kubernetes.io/os: linux
 
+- name: Wait for HostPathProvisioner status to become Available
+  kubernetes.core.k8s_info:
+    api_version: hostpathprovisioner.kubevirt.io/v1beta1
+    kind: HostPathProvisioner
+    name: hostpath-provisioner
+  register: hostpath_status
+  until: >
```

Comment:
Would it be possible to use `wait_condition` instead?

https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_info_module.html#parameter-wait_condition

Reply:
added and tested